cfme/physical/provider/__init__.py

```diff
@@ -2,13 +2,21 @@
 from widgetastic.utils import Fillable
 
 from cfme.base.ui import Server
+
+from cfme.utils import version
 from cfme.common.provider import BaseProvider
-from cfme.common.provider_views import PhysicalProviderAddView, PhysicalProvidersView
+from cfme.common.provider_views import (PhysicalProviderAddView,
+                                        PhysicalProvidersView,
+                                        PhysicalProviderDetailsView,
+                                        PhysicalProviderEditView)
 from cfme.utils.appliance import Navigatable
 from cfme.utils.appliance.implementations.ui import navigator, CFMENavigateStep
 from cfme.utils.pretty import Pretty
 from cfme.utils.varmeth import variable
+from cfme.utils.appliance.implementations.ui import navigate_to
 
+from cfme.utils.log import logger
+from cfme.fixtures import pytest_selenium as sel
```

Comment:
This module should not need to be imported, and is being deprecated.

In this case its only used for its NoSuchElementException, which you can import from widgetastic.exceptions.